cpu/stm32f1/periph/adc.c

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright (C) 2015 Engineering-Spirit
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     cpu_stm32f1
+ * @{
+ *
+ * @file
+ * @brief       Low-level ADC driver implementation
+ *
+ * @author      Nick van IJzendoorn <nijzendoorn@engineering-spirit.nl>
+ *
+ * @}
+ */
+
+#include "cpu.h"
+#include "periph/adc.h"
+#include "periph_conf.h"
+
+/* guard in case that no ADC device is defined */
+#if ADC_NUMOF
```

Comment:
Where do you define those preprocessor macros? On which board do you test the driver? As far as I see it, the only RIOT supported boards with a stm32f1 CPU are iotlab-m3, spark-core and fox. But you did not add peripheral configurations for any of them